src/view.c

```diff
@@ -1667,9 +1668,10 @@ append_line_format(struct view *view, struct line *line, const char *fmt, ...)
 	if (fmtlen <= 0)
 		return false;
 
-	box = realloc(box, box_sizeof(box, 0, fmtlen));
-	if (!box)
+	temp_box = realloc(box, box_sizeof(box, 0, fmtlen));
```

Comment:
‘box’ capture the value in ‘line->data’, so this assignment is not supposed to lose anything.